### DISCUSSION TITLE
Jon Lawrence MapReader runthrough

### DISCUSSION THREAD

**Discussion by kmcdono2** [maps-as-data/MapReader#143] [General]
# 28 March 2023 Run through with @jl106jml 

### Installation instructions
- add language about using "terminal" in Windows/Anaconda prompt?
- git for windows needs installing
- how to use EDINA Digimap data (e.g. transform single-channel images to png)

### Load & Patchify
- separate "Further analysis & Viz" as "exploring inputs" documentation distinct from Load & Patchify. This needs to be clear to ensure people understand what HAS to be done vs what is interesting to see. Add comment about you SHOULD do this if you have lower-quality images. But for those with high-quality images (and define this), this is optional.

## Annotation task
- add patch in context of whole map sheet (with red bounding box for patch visible) as 2nd context image for patch during annotation?
- "[WARNING]" re: mean_pixel_RGB should be a NOTE (not to dissuade people from next "annotate" command)
- include note early on in setting up annotation task about code to SAVE annotations

### Learn & Predict
- start all counts at 0 (see command below for ex)
- `annotated_images.adjust_label(shiftby=-1)` should be `annotated_images.adjust_labels(shiftby=-1)`
- document that it's possible to select patches to review by label number (but also check that this actually functions).

**Comment by rwood-97** [comment]
Bug when downloading maps (``download`` subpackage) in that some maps are only partially downloaded or have missing sections which come up as blue areas?

@kmcdono2 will you add screenshots?
